VideoWeb/VideoWeb/ClientApp/src/app/shared/video-filter/video-filter.component.scss

```diff
@@ -0,0 +1,96 @@
+@import '~govuk-frontend/govuk/base';
+
+.v-bg-panel {
+  padding: 12px 12px 25px 12px;
+  margin-top: 8px;
+  overflow: hidden;
+  border: solid 1px #eee;
+  border-radius: 3px;
+  background: #fefefe;
+  box-shadow: inset 0px 0px 2px rgb(0 0 0 / 10%);
+  box-shadow: inset 0px 0px 8px rgb(0 0 0 / 5%);
+}
+
+.bg-option-text {
+  font-size: 1.1em;
+  margin-bottom: 4px;
+}
+
+.bg-option-img {
+  width: 120px;
+  cursor: pointer;
+}
+
+.vh-accordion {
+  font-family: 'GDS Transport', Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 1.2em;
+  background-color: govuk-colour('white');
+  color: govuk-colour('black');
+  cursor: pointer;
+  width: 100%;
+  border: none;
+  text-align: left;
+  outline: none;
+  transition: 0.4s;
+}
+
+.input-hidden {
+  position: absolute;
+  left: -9999px;
+}
+
+input[type='radio']:checked + label > img {
+  border: 4px solid govuk-colour('yellow');
+}
+
+//Grid for background images
+.bg-grid {
```

Comment:
I'd prefer if bg was background. 

How does this grid look on smaller screens? I'd also wonder about if other options are added, they will have to be added here as well, might be a way other than grid to do.

Reply:
It should be fine, I used the styles provided also develop with the minimum VH size requirements of 1366 x 768